MorkoUnityProject/Assets/Scripts/ObservableList.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
```

Comment:
Add header, aka own name and purpose of file/class in an comment at the top of file like
/*
Coder Champion

This is a class for containing abandoned souls
*/